components/common/actionButton/deletePopover.tsx

```
@@ -32,14 +37,14 @@ const DeletePopover: FC<DeletePopoverProps> = ({
           <Row gutter={20}>
             {onCancel && (
               <Col span={12}>
-                <Button
-                  id={`buttonPopOverActionFor${titleText}Id`}
-                  size="small"
-                  className={buttonClass}
-                  onClick={onCancel}
-                >
-                  Cancel
-                </Button>
+                                 <Button
```

ESLint

[error] Delete `·················`

(prettier/prettier)